glean-core/android/build.gradle

```diff
@@ -118,13 +121,17 @@ android {
 }
 
 dependencies {
+    api "org.mozilla.components:concept-fetch:$android_components_version"
+    implementation "org.mozilla.components:lib-fetch-httpurlconnection:$android_components_version"
```

Comment:
So...  We've grown this dependency on `android-components`.  Maybe there's no harm (in a separate PR) from depending on the logging component as well, which would give us better syntax and the ability to redirect logcat to stdout etc...

Reply:
I'd agree with you in general. However, this would help us with showing the logs from the Rust part (which are using Android Log). See [this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1551595)